src/system-health/setup.py

```diff
@@ -0,0 +1,42 @@
+from setuptools import setup
+
+setup(
+    name='system-health',
+    version='1.0',
+    description='SONiC system health package',
+    license='Apache 2.0',
+    author='SONiC Team',
+    author_email='linuxnetdev@microsoft.com',
+    url='https://github.com/Azure/sonic-buildimage',
+    maintainer='Junchao Chen',
+    maintainer_email='junchaow@mellanox.com',
+    packages=[
+        'health_checker',
+        'tests'
+    ],
+    scripts=[
+        'scripts/healthd',
+    ],
+    setup_requires= [
+        'pytest-runner'
+    ],
+    tests_require = [
```

Comment:
Can you add an `install_requires` section here and list all the wheels needed at runtime? E.g.,
```
      install_requires=[
          'sonic-py-common',
          'swsssdk>=2.0.1'
      ],
```

Reply:
Sure.